bundle/Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryserviceplugin.js

```diff
@@ -297,6 +298,83 @@ YUI.add('cof-createcontent-universaldiscoveryserviceplugin', function (Y) {
         _deleteContent: function (event) {
             event.content.delete({api: this.get('host').get('capi')}, function () {});
         },
+
+        /**
+         * changeLanguage event handler. It opens languageSelectionBox for selecting
+         * language of created content
+         *
+         * @method _selectLanguage
+         * @param e {Object} event facade
+         */
+        _selectLanguage: function (e) {
+            var host = this.get('host');
+
+            e.preventDefault();
+
+            /**
+             * Fired to opens languageSelectionBox for selecting language of created content
+             * Listened in the eZ.Plugin.LanguageSelectionBox
+             *
+             * @event languageSelect
+             * @param config {object} the configuration of language selection box
+             */
+            host.get('app').fire('languageSelect', {
+                config: {
+                    title: Y.eZ.trans('change.language.to', {}, 'contentedit'),
+                    languageSelectedHandler: Y.bind(this._setLanguage, this, e.target, e.fields),
+                    cancelLanguageSelectionHandler: null,
+                    canBaseTranslation: false,
+                    translationMode: true,
+                    referenceLanguageList: [host.get('languageCode')]
+                },
+            });
+        },
+
+        /**
+         * Sets language of created content to one given in event facade. After that notification is fired.
+         *
+         * @method _setLanguage
+         * @private
```

Comment:
`@protected` 
Making it private is a bit misleading in the context of YUI views.